### PR TITLE
Screenshots go through

### PR DESCRIPTION
**Pull request by anenadic**
Includes:
- modifications to 'alt' text of images
- use of Markdown syntax for including images instead of HTML
-  use of image-with-shadow class 
- renaming of some of the images
- fix for #253

### CODE REVIEW

File: _episodes/introduction.md [hunk 1]
```diff
@@ -288,16 +292,16 @@ your own website.
 > for more information about possible licenses.
 {: .callout}
 
-<img src="../fig/set_CC0_license.png" alt="Repository name set to group-website" width="400">
+![Set licence to CC0](../fig/set_CC0_license.png){: .image-with-shadow width="400px" }
```
Comment (tobyhodges) [inline _episodes/introduction.md]:
```suggestion
![Set licence to CC0](../fig/set_CC0_license.png){: .image-with-shadow width="600px" }
```

this was too small to read properly on my screen.

**Review by tobyhodges — COMMENTED** [review]
Looks good except for one screenshot that was too small. The shadows on the images really help!

**Comment by tobyhodges** [comment]
I pushed a few updated versions of screenshots. One more to do and then I think this is ready to go!

**Comment by tobyhodges** [comment]
Thank you @anenadic!

**Comment by anenadic** [comment]
> Looks good except for one screenshot that was too small. The shadows on the images really help!

I really like the shadows too - they make quite a difference.

> Thank you @anenadic!

And you @tobyhodges! 🎉  Will delete this branch now.

**Pull request merged**